### DISCUSSION TITLE
Documentation fix request: Beats encryption

### DISCUSSION THREAD

**Discussion by jaredatobe** [Security-Onion-Solutions/securityonion#4964] [Ideas] [answered]
Several people have noticed a mistake in the [2.3 documentation for Beats encryption](https://docs.securityonion.net/en/2.3/beats.html#encryption).

https://github.com/Security-Onion-Solutions/securityonion/discussions/2503
https://github.com/Security-Onion-Solutions/securityonion/discussions/3692
https://github.com/Security-Onion-Solutions/securityonion/discussions/4652#discussioncomment-955149
https://github.com/Security-Onion-Solutions/securityonion/discussions/4946

The correct information:

Put the certs and key in the following directory: /opt/so/saltstack/local/salt/logstash/etc/certs

Then run "so-logstash-restart" and the files will be placed permanently in "/opt/so/conf/logstash/etc/certs" and can be accessed in the docker container of logstash via "/usr/share/logstash/certs".

**Comment by weslambert** [comment, marked as answer]
This has been updated: https://github.com/Security-Onion-Solutions/securityonion-docs/commit/32c2410cb574cc958b0c3192ec18a87233522768

https://docs.securityonion.net/en/latest/beats.html?highlight=beats#using-the-beats-default-port-5044-with-encryption
